std/socket.d

```diff
@@ -2041,14 +2041,23 @@ static if (is(sockaddr_un))
     @safe unittest
     {
         import core.stdc.stdio : remove;
-        import std.file : deleteme;
+
+        static string deleteme()
```

Comment:
A short comment explaining the need for this local definition would be nice.

Reply:
Done. It's in the commit message as well.